services/codeclimate/codeclimate-coverage.service.js

```diff
@@ -53,15 +53,19 @@ export default class CodeclimateCoverage extends BaseJsonService {
   }
 
   async fetch({ user, repo }) {
+    const repoInfos = await fetchRepo(this, { user, repo })
+    const repoInfosWithTestReport = repoInfos.filter(
+      repoInfo => repoInfo.relationships.latest_default_branch_test_report.data
+    )
+    if (repoInfosWithTestReport.length === 0) {
```

Comment:
Generally curious what the response would represent if there were multiple entries that had test reports :man_shrugging:

Reply:
We'll know when and if a future user complains! 😄